ipmininet/router/config/bgp.py

```diff
@@ -306,8 +299,9 @@ def filters_to_match_cond(self,
                               filter_list: Sequence[Union[AccessList,
                                                           CommunityList,
                                                           PrefixList]],
-                              family):
+                              family=None):
```

Comment:
Why setting it to None if you only want either 'ipv4' or 'ipv6' ?

Reply:
Oops, this this was due to an oversight in a local refactor. Changed in dca805acddbe19863aea03c4ed488785ca0b650e